runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/ChunkingShuffleBatchReader.java

```diff
@@ -17,18 +17,18 @@
  */
 package org.apache.beam.runners.dataflow.worker;
 
-import java.io.ByteArrayInputStream;
+import com.google.protobuf.ByteString;
```

Comment:
The worker doesn't depend on protobuf directly.

Its probably best if we use the vendored grpc import paths of protobuf